CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
-
 # Changelog
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.3] - 2023-02-23
+
+### Added
+ - Add 403 invalid token error response
+ - Make it possible to increase MaxSessionLifetime without increasing the SessionResult lifetime
+ - Make email recovery link expiry configurable
+
+### Fixed
+ - Dependency version bumps
```

Comment:
This is not a 'Fixed', but 'Security' and then we should better explain what we fixed.